web/src/documents.ts

```diff
@@ -210,14 +215,19 @@ export async function cancel(
 }
 
 /**
- * Restart the document's kernel space
+ * Restart one, or all, of the kernels in a document's kernel space
+ *
+ * If `kernelId` is not supplied then all kernels in the kernel space
+ * will be restarted.
```

Comment:
@alex-ketch This is the first of several changes I'll make here: now all, or a specific, kernel in the document's kernel space will be restarted.